lib/install/validate/config_to_data.go

```diff
@@ -23,7 +23,7 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/go-units"
 
-	"path/filepath"
+	"path"
```

Comment:
Go imports should have three groups - see https://github.com/vmware/vic/pull/5865#discussion_r130988908.